# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/platforms/ffmpeg/FFmpegRDFTTypes.h
# Commit: 9b4ea6ecadd1
# Full Hash: 9b4ea6ecadd1e56b9a279feab220a8bfab36809c
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-02-28 16:14:38
# Regressor Bug: 1879873
# File Overlap Count: 1
# Description:
#   Bug 1879873 - Directly use tx instead of the older avfft interface. r=karlt,media-playback-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D202156
# ==============================================================================

diff -r 7c71c3d1a738 -r 9b4ea6ecadd1 dom/media/platforms/ffmpeg/FFmpegRDFTTypes.h
--- a/dom/media/platforms/ffmpeg/FFmpegRDFTTypes.h	Wed Feb 28 12:50:27 2024 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,34 +0,0 @@
-/* -*- Mode: C++; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
-/* vim: set ts=8 sts=2 et sw=2 tw=80: */
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
-
-#ifndef FFmpegRDFTTypes_h
-#define FFmpegRDFTTypes_h
-
-struct RDFTContext;
-
-using FFTSample = float;
-
-enum RDFTransformType {
-  DFT_R2C,
-  IDFT_C2R,
-  IDFT_R2C,
-  DFT_C2R,
-};
-
-extern "C" {
-
-using AvRdftInitFn = RDFTContext* (*)(int, enum RDFTransformType);
-using AvRdftCalcFn = void (*)(RDFTContext*, FFTSample*);
-using AvRdftEndFn = void (*)(RDFTContext*);
-}
-
-struct FFmpegRDFTFuncs {
-  AvRdftInitFn init;
-  AvRdftCalcFn calc;
-  AvRdftEndFn end;
-};
-
-#endif  // FFmpegRDFTTypes_h